src/components/Gyoithon/Gyoithon.tsx

```diff
@@ -18,28 +24,42 @@ interface FormValues {
     export_path: string;
```

Comment:
Change select_file, import_file and export_path to camel case, i.e. selectFile, importFile and exportPath